src/lib/server/orders.ts

```diff
@@ -854,13 +855,12 @@ export async function createOrder(
 			...(params.engagements && { engagements: params.engagements })
 		};
 		await collections.orders.insertOne(order, { session });
-		if (order.status === 'paid' && paymentMethod && paymentMethod === 'free') {
-			await updateAfterOrderPaid(order, session);
-		}
-		if (paymentMethod && paymentMethod !== 'free') {
+
+		let orderPayment;
```

Comment:
Please type it correctly. Currently it's type `any`.

```suggestion
		let orderPayment: ... | undefined = undefined;
```